pkgs/data/fonts/nerd-fonts/convert-license.nix

```
@@ -0,0 +1,28 @@
+{ lib }:
+let
+  ls = lib.licenses;
+in
+licenseString:
```

And return a function:

```suggestion
(licenseString:
```